sdk/documentintelligence/azure-ai-documentintelligence/README.md

```diff
@@ -896,6 +897,74 @@ The following add-on capabilities are available in this SDK:
 
 Note that some add-on capabilities will incur additional charges. See pricing: https://azure.microsoft.com/pricing/details/ai-document-intelligence/.
 
+### Get Raw JSON Result
+
+Can get the HTTP response by passing parameter `raw_response_hook`.
+<!-- SNIPPET:sample_get_raw_response.raw_response_hook -->
+
+```python
+import os
+from azure.core.credentials import AzureKeyCredential
+from azure.ai.documentintelligence import DocumentIntelligenceAdministrationClient
+endpoint = os.environ["DOCUMENTINTELLIGENCE_ENDPOINT"]
+key = os.environ["DOCUMENTINTELLIGENCE_API_KEY"]
+
+client = DocumentIntelligenceAdministrationClient(
+    endpoint=endpoint, credential=AzureKeyCredential(key)
+)
+
+responses = []
+
+def callback(response):
+    response_status_code = response.http_response.status_code
+    response_body = response.http_response.json()
+    responses.append(response_status_code)
+    responses.append(response_body)
```

Comment:
Maybe this should be a dict so that the status code is more closely linked to the response body?

Reply:
No, the response body is a dict of analyzed result, not including `status_code`. For eample, the response body in this sample is:
```
{'customDocumentModels': {'count': 649, 'limit': 20000}, 'customNeuralDocumentModelBuilds': {'used': 0, 'quota': 20, 'quotaResetDateTime': '2024-12-01T00:00:00Z'}}
```

Reply:
I meant do something like:
```
{
  "status": <status code here>,
  "response": <response body here>
}
```

Reply:
Updated